versions.props

```diff
@@ -1,7 +1,7 @@
 org.slf4j:* = 1.7.25
 org.apache.avro:avro = 1.10.1
 org.apache.calcite:* = 1.10.0
```

Comment:
Do we know if our calcite version needs to be upgraded?

Reply:
The apache flink runtime won't depend on this calcite jar ,  this calcite was introduced for [hive](https://github.com/apache/iceberg/pull/2110/files) before.  So I think we don't  need to change this calcite version for flink upgrading.